demo/billing/mzcompose.yml

```diff
@@ -84,6 +87,13 @@ volumes:
 
 mzconduct:
   workflows:
+    local:
```

Comment:
I'd consider renaming this to `demo` to make it obvious to future developers that this is what shows up in the docs.